package.json

```diff
@@ -5,7 +5,7 @@
     "node": "^10"
   },
   "scripts": {
-    "dev": "next ./ -p 3001",
+    "dev": "next ./ -p 3000",
```

Comment:
that was on 3001 as `now dev` was on 3000

Reply:
Would the be run at the same time? I figured keep the ports the same for cookies, etc.

Reply:
prolly not. I've done it a few times to compare a production output vs the now dev run.  but cookies is also a good reason.